mapflow/mapflow.py

```diff
@@ -1792,6 +1800,23 @@ def submit_processing_rating_callback(self, response: QNetworkReply) -> None:
         )
         self.update_processing_current_rating()
 
+    def enable_rating_submit_button(self, radio_checked: Optional[bool]) -> None:
+        status_ok = False
+        radio_button_checked = False
+        row = self.dlg.processingsTable.currentRow()
+        if not row >= 0:
+            return
+        if self.dlg.processingsTable.item(row, self.config.PROCESSING_TABLE_STATUS_COLUMN_INDEX).text() == 'OK':
+            status_ok = True
+        for radio_button in self.radio_buttons:
```

Comment:
isn't there a QT-native way to know which radiobutotn is checked in the group? 

If not, you can use more pythonic  `any(radio_button.checked() for radio_button in self.radio_buttons)`

Reply:
Fixed